src/Terrain/TerrainType.java

```diff
@@ -161,9 +163,10 @@ private TerrainType(int attributeFlags, int defense, Color mainColor, String nam
   public static final TerrainType SHOAL = new TerrainType( SHOAL_FLAGS, SHOAL_DEFENSE, SHOAL_COLOR, SHOAL_NAME );
 
   // List of all terrain types.
-  public static final TerrainType[] TerrainTypeList = {
-    AIRPORT, BRIDGE, CITY, BUNKER, PILLAR, DUNES, FACTORY, FOREST, GRASS, HEADQUARTERS, LAB, MOUNTAIN, REEF, RIVER, ROAD, SEA, SEAPORT, SHOAL
-  };
+  public static final ArrayList<TerrainType> TerrainTypeList =
```

Comment:
This feels a little bit like re-introducing the `TerrainType` enum we killed a while back.